crates/bitwarden/src/mobile/vault/ciphers/cipher_decrypt.rs

```diff
@@ -5,12 +5,14 @@ use crate::vault::{Cipher, CipherView};
 
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "mobile", derive(uniffi::Record))]
 pub struct CipherDecryptRequest {
-    pub cipher: Box<Cipher>,
+    pub cipher: Cipher,
```

Comment:
@dani-garcia I had to remove these box to have uniffi happy. I didn't want to implement custom transformers for everything. Not sure if there is another way to better handle this.

Reply:
The box here is mostly to fix Clippy's complaining about unbalanced enums, seems like we can just change `` in `bitwarden-json` to be defined like this:
```rust
pub enum MobileCiphersCommand {
    Encrypt(Box<CipherEncryptRequest>),
    Decrypt(Box<CipherDecryptRequest>),
    DecryptList(CipherDecryptListRequest),
}
```
And then just using `*cmd` to access the inner value of the box when matching against the commands, that seems to remove the clippy lint to me at least.